### PR TITLE
Update responsable disclosure

### LINKED ISSUE #1374: Chore: update responsible disclosure policy
### Describe the change
Update the responsible disclosure policy to include:
- scopes
- allowed attack methods
- disallowed attack methods
- pgp key

### Motivation
To make it easier to understand what is allowed.

### Current implementation
https://thalia.nu/responsible-disclosure/
https://github.com/svthalia/concrexit/blob/master/website/singlepages/templates/singlepages/responsible_disclosure.html

### PR DESCRIPTION
**Pull request by Luca-Castelnuovo**
Closes #1374 

### Summary
Update the responsible disclosure

### How to test
Steps to test the changes you made:
1. Go to 'localhost/responsible-disclosure/'

### CODE REVIEW

**Comment by Luca-Castelnuovo** [comment]
todo: verkeerde pgp sleutel

**Comment by Legosteen11** [comment]
deploy review

File: website/singlepages/templates/singlepages/responsible_disclosure.html [hunk 1]
```diff
@@ -37,13 +40,39 @@ <h4>{% trans "We promise you to" %}</h4>
                 <li>{% trans "take your reports seriously." %}</li>
                 <li>{% trans "respond to your reports as soon as possible." %}</li>
                 <li>{% trans "solve the issue as soon as possible." %}</li>
-                <li>{% trans "respect your privacy and will treat your reports confidentially." %}</li>
+                <li>{% trans "respect your privacy and we will treat your reports confidentially." %}</li>
```
Comment (Legosteen11) [inline website/singlepages/templates/singlepages/responsible_disclosure.html:43]: Do you think 'we' should be added here?

**Review by Legosteen11 — DISMISSED** [review]

**Comment by Legosteen11** [comment]
remove review

File: website/singlepages/templates/singlepages/responsible_disclosure.html (hunk 1 above)
Comment (mark-boute) [reply website/singlepages/templates/singlepages/responsible_disclosure.html:43]: I think 'will' should be removed

"... privacy and treat your ..."

Comment (Luca-Castelnuovo) [reply website/singlepages/templates/singlepages/responsible_disclosure.html]: I updated the file.

**Review by mark-boute — APPROVED** [review]

**Pull request merged**